h2/events.py

```diff
@@ -136,6 +136,51 @@ def __repr__(self):
         )
 
 
+class ResponseSent(object):
```

Comment:
If the answer is that we probably do, can we replace both with a singular `HeadersSent`? When `TrailersSent` exists, I think those two names are clear enough.